### PR TITLE
Global Screen  - render StartupPages in ilGlobalPageTemplate

### PR DESCRIPTION
**Pull request by nhaagen**
ilStartUpGUI::initStartUpTemplate now returns a template to set content at (instead of directly working on global $tpl).
ilStartUpGUI::printToGlobalTemplate will then print the contents to ilGlobalPageTemplate.

@chfsx I call ilGlobalPageTemplate::printToStdout with a_skip_main_menu=true - there still is a mainbar, however. This feels odd, but probably I'm missing something.
Will we have the mainbar for a public section?

### CODE REVIEW

**Comment by chfsx** [comment]
Hi @nhaagen 
Thanks you very much for the PR! I made some modifications and opened a PR against your branch. This will fix the issue with the MainBar, but first PR #2117 is needed for this.

@smeyer-ilias are you fine with the changes in your Component?

**Comment by nhaagen** [comment]
Hey @chfsx ,
thanks a lot for weeding out even more. I like the tpl.main.html  ;)

File: Services/Init/templates/default/tpl.startup_screen.html [hunk 1]
```diff
@@ -16,6 +16,15 @@ <h5 class="ilAccHeadingHidden"><a id="il_message_focus" name="il_message_focus">
 	</div>
 	<!-- END success -->
 </div>
+
```
Comment (chfsx) [inline Services/Init/templates/default/tpl.startup_screen.html:19]: @nhaagen das müsste vermutlich anders gelöst werden, ich würde dies als Topbar-item in der neuen `StartUpModificationProvider` machen

Comment (nhaagen) [reply Services/Init/templates/default/tpl.startup_screen.html]: Eine LanguageSelection-Slate?

**Comment by Amstutz** [comment]
Congrats to @chfsx and @nhaagen :

grafik
This is where we should aim at! Nice work.

**Comment by chfsx** [comment]
@smeyer-ilias on my side it's all good, are there any objections/supplements from you? We'd love to merge this this week, if that's okay with you.

**Comment by chfsx** [comment]
I discussed this PR with @smeyer-ilias via email and merged locally with some adjustments.

**Comment by chfsx** [comment]
Thanks @nhaagen for the implementation!

**Pull request closed without merging**